packages/react-native-skia/components/image/remoteImage.js

```diff
@@ -0,0 +1,111 @@
+/*
+https://pngimg.com/uploads/lion/lion_PNG23269.png
+https://pngimg.com/uploads/tiger/tiger_PNG23244.png
+https://pngimg.com/uploads/fish/fish_PNG25146.png
+https://pngimg.com/uploads/arctic_fox/arctic_fox_PNG41376.png
+https://pngimg.com/uploads/panda/panda_PNG10.png
+https://pngimg.com/uploads/gorilla/gorilla_PNG18708.png
+https://pngimg.com/uploads/mango/mango_PNG9173.png
+https://pngimg.com/uploads/birds/birds_PNG95.png
+*/
+import * as React from 'react';
+import { View, AppRegistry, Image, Text } from 'react-native';
+
+const SimpleViewApp = React.Node = () => {
+  return (
+   <>
+      <View
+          style={{
+                  flex:0.5,
+                  flexDirection:'row',
+          }}>  
+        <View
+            style={{ width: 250,
+                     height: 250,
+                     marginLeft:10,
+                     backgroundColor:'cornsilk',
+            }}>
+            <Image
```

Comment:
The test app to include below use cases 
Case-1. : Network source images 
Case-2 :  Local source images
Case-3 : Network + Local source images 
Case-4 : Dynamic update of image path for Case-3 

Please add the missing cases in the same test app and verify.